lib/screens_web/router.ex

```diff
@@ -79,12 +85,17 @@ defmodule ScreensWeb.Router do
   end
 
   scope "/v2", ScreensWeb.V2 do
+    scope "/widget" do
+      pipe_through [:redirect_prod_http, :browser_no_csrf]
+      post "/:app_id", ScreenController, :widget
+      get "/:app_id", ScreenController, :widget
+    end
```

Comment:
I needed to put the endpoint in a separate scope in order to apply a different set of plugs/pipelines to it.
This was the best naming I could come up with; I'd be happy to go with a different one!